init_friendbot.go

```diff
@@ -57,15 +57,19 @@ func initFriendbot(cfg Config, secrets Secrets) (*internal.Bot, error) {
 		return nil, errors.Wrap(err, "creating minion accounts")
 	}
 	log.Printf("Adding %d minions to friendbot", len(minions))
-	return &internal.Bot{Minions: minions}, nil
+	return &internal.Bot{
+		Minions:               minions,
+		NetworkClient:         networkClient,
+		FundContractAddresses: cfg.FundContractAddresses,
```

Comment:
If FundContractAddresses=true and NetworkClient = Horizon, is this invalid config state? Shiould Friendbot just panic to alert operator?